src/commands/getAddCodeToChatCommand.ts

```diff
@@ -1,18 +1,17 @@
 import * as vscode from "vscode";
-import { ChatManager } from "../Chat/ChatManager";
-import { ChatWebviewProvider } from "../providers/ChatViewProvider";
-import { addSelectedCodeToChat } from "../helpers/addSelectedCodeToChat";
-
-/**
- * Command to add the selected code (or whole file if no selection) to the active chat.
- * Opens webview and/or starts chat if necessary.
- */
+import { ChatEditor } from "../Chat/ChatEditor";
+import { ChatDataManager } from "../Chat/ChatDataManager";
+import { addCodeToChat } from "../helpers/addCodeToChat";
 
 export const getAddCodeToChatCommand = (
-  chatWebviewProvider: ChatWebviewProvider,
-  chatManager: ChatManager,
+  chatEditor: ChatEditor,
+  chatDataManager: ChatDataManager,
 ) =>
   vscode.commands.registerCommand("vscode-byolad.addCodeToChat", async () => {
-    addSelectedCodeToChat(chatManager, chatWebviewProvider);
-    chatWebviewProvider.updateChat(chatManager.chats, chatManager.activeChatId);
+    const chat = chatDataManager.getActiveChat();
+    if (!chat) {
+      vscode.window.showErrorMessage("No active chat");
```

Comment:
I guess I forgot there was an "Add code to new chat" command, so I guess it's more of a preference to have the command add to the active chat and, if no active chat, create one.

Reply:
Yeah in theory the "Add code to chat" command shouldn't be available unless there is an active chat and vice versa with the "Add code to new chat" command. Though it might better to have "Add code to chat" just start a new chat if for some reason it is called and there isn't an active one, as a failsafe?